src/main/deploy/swerve/modules/physicalproperties.json

```diff
@@ -11,7 +11,7 @@
   },
   "conversionFactor": {
     "angle": 19.2,
-    "drive": 0.0696793458626792
+    "drive": 0.06658958893604663608562691131498
```

Comment:
Floats only have 7 significant digits (plus an exponent) and doubles have 16 decimal digits (plus an exponent) of precision, so this may be overkill.